maintainers/team-list.nix

```diff
@@ -134,6 +134,21 @@ with lib.maintainers;
     shortName = "c3d2";
   };
 
+  categorization = {
+    members = [
+      aleksana
+      fgaz
+      getpsyched
+      lyndeno
+      natsukium
+      pyrotelekinetic
```

Comment:
I could not find an entry in `maintainer-list.nix` for this person.

Reply:
I removed @pyrotelekinetic per above. Could you add yourself to `maintainers/maintainer-list.nix`?

Reply:
PR'd: https://github.com/NixOS/nixpkgs/pull/362952

Reply:
Rebased and readded to the team list. Thank you ❤️